flink-end-to-end-tests/pom.xml

```diff
@@ -89,6 +89,16 @@ under the License.
 		<module>flink-end-to-end-tests-hbase</module>
 	</modules>
 
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>net.java.dev.jna</groupId>
```

Comment:
This is for dependency convergence, right? So we should add a comment to say for which projects in case someone is wondering in the future.